server/parsec/client_context.py

```diff
@@ -173,6 +173,8 @@ class AnonymousAccountClientContext:
     client_api_version: ApiVersion
     settled_api_version: ApiVersion
     logger: ParsecBoundLogger = field(init=False)
+    user_agent: str
+    user_ip: str
```

Comment:
Since ASGI doesn't guarantee that the ip can be provided, we should type it as `str | None` and add a comment (typically linking to https://asgi.readthedocs.io/en/latest/specs/www.html#http-connection-scope)